test/test_contrib.py

```diff
@@ -387,8 +387,8 @@ class TestImageStitcher:
     @pytest.mark.parametrize("estimator", ['ransac', 'vanilla'])
     def test_smoke(self, estimator, device, dtype):
         B, C, H, W = 1, 3, 224, 224
-        input1 = torch.rand(B, C, H, W, device=device, dtype=dtype)
-        input2 = torch.rand(B, C, H, W, device=device, dtype=dtype)
+        sample1 = torch.rand(B, C, H, W, device=device, dtype=dtype)
```

Comment:
in this case not really needed since the main issue is with `input` -- but in the end your proposal doesn't harm

Reply:
I thought it would be more consistent to also rename these.

Reply:
yeah -- it is !